user/views.py

```diff
@@ -133,5 +137,23 @@ def get(self, request):
         except KeyError:
             return JsonResponse({"message":"KEY_ERROR"}, status=400)
 
+class SMSCheckView(APIView):
+    def post(self, request):
+        try:
+            p_num = request.data['phone_number']
+        except KeyError:
+            return Response({'message': 'Bad Request'}, status=400)
+        else:
+            SMSAuthRequest.objects.update_or_create(phone_number=p_num)
+            return Response({'message': 'OK'})
```

Comment:
승진님 ! try except else 너무 잘 써주셨네요! :) 
절차 지향적인 코드로, 에러가 날 수 있는 상황을 잘 캐치하신 것 같습니다.
하나 추천드리자면, 파이썬에서는 코드 가운데에서 exception을 처리하는 스타일을 선호하지 않는 편입니다.
처리하는 에러가 KeyError 하나뿐일 때에는 더욱, 큰 에러가 없을 때 자연스러운 흐름을 따라 코드가 읽히도록 하는 것을 지향하고 있습니다.

```suggestion
        try:
            p_num = request.data['phone_number']
            SMSAuthRequest.objects.update_or_create(phone_number=p_num)
            return Response({'message': 'OK'})
        except KeyError:
            return Response({'message': 'PHONE_NUMBER_REQUIRED'}, status=400)
```

Reply:
네 반영하겠습니다. 감사합니다 :)